scripts/sts/sts.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# USAGE
+#
+# ~ Creation ~
+#
+# make ocm/provision_sts (optional <CLUSTER_NAME= > <AWS_REGION= > <NODE_COUNT= > <MACHINE_TYPE= >)
+#
+# <Default values>
+#  CLUSTER_NAME=defaultsts
+#  AWS_REGION=eu-west-1
+#  NODE_COUN=4
+#  MACHINE_TYPE=m5.xlarge
+#
+# ^C to break
+# Create a STS cluster
+#
+# ~ Deletion ~
+#
+# make ocm/delete_sts (optional <CLUSTER_NAME= > <AWS_REGION= > <PREFIX= >)
+#
+# <Default values>
+#  PREFIX=ManagedOpenShift
+#
+# ^C to break
+# Delete a STS cluster
+#
+# PREREQUISITES
+# - ROSA CLI
+# - AWS CLI
+# - jq
+# - aws configuration with valid permissions. Run `aws configure`
+#
+# MORE INFO
+# https://docs.openshift.com/rosa/rosa_getting_started_sts/rosa-sts-setting-up-environment.html
+
+set -eux
+
+# Prevents aws cli from opening editor on responses - https://github.com/aws/aws-cli/issues/4992
+export AWS_PAGER=""
+ROLE_NAME="rhoam_role"
+OCM_ENV="${OCM_ENV:-production}"
+LOCAL_RUN=${LOCAL_RUN:-true}
```

Comment:
```suggestion
OCM_ENV="${OCM_ENV:-production}"
```
`LOCAL_RUN` is not used anywhere I think :thinking: